README.md

```diff
@@ -3,32 +3,31 @@
 
 # Eclipse Che plugin registry
 
-## Build Eclipse Che plugin registry docker image
+## Build Eclipse Che plugin registry container image
+
+Most of the time you won't need to rebuild the image because we build ```quay.io/openshiftio/che-plugin-registry:latest``` after every commit in master. In case you needed to change the content of the registry (e.g. add or modify some plugins meta.yaml) you can build your own image executing
```

Comment:
hello, it seems we are deprecating this image name: 
https://github.com/eclipse/che-devfile-registry/pull/31

Reply:
yes

Reply:
yes, once that PR is merged we will do another one for this registry and another PR to move from `openshiftio` org to `eclipse-che`.